setup.py

```diff
@@ -16,6 +16,22 @@
 except ImportError:
     tagger = egg_info_cmd.egg_info
 
+install_requires=[
+    'requests',
+    'PyYAML',
+    'rdflib >= 4.1.0',
+    'rdflib-jsonld >= 0.3.0',
+    'mistune'
+]
+
+avro_package = 'avro'
+if sys.version_info >= (3,):
+    avro_package += '-python3'
+install_requires.append(avro_package)
+
+if sys.version_info >= (2, 7):
+    install_requires.append('typing')
+
```

Comment:
Thanks for the reminder :-) We did a similar thing in khmer https://github.com/dib-lab/khmer/blob/master/setup.py#L232